src/app/profile/page.tsx

```diff
@@ -6,16 +6,21 @@ import { ThemeToggler } from '@/components/ui/theme-toggler'
 
 export default function Profile() {
   return (
-    <AuthContent>
-      <ImageCard className="-my-[4.375rem] hidden sm:flex py-32 px-[3.25rem]">
+    <AuthContent className="profile-page bg-white md:bg-gradient-blue-white">
```

Comment:
We have white color in our design system. Also, use section here now with some padding

```suggestion
    <AuthContent className="bg-base-white md:bg-gradient-blue-white">
      <section className="flex-col md:flex-row justify-center pt-3 grow md:grow-0 flex gap-7 rounded-3xl md:dark:bg-base-black md:py-10 md:px-[1.9375rem] md:bg-base-white lg:gap-16">
```